modules/publicize/ui.php

```diff
@@ -263,12 +273,17 @@ function admin_page() {
 
 
 							<?php
-								$connections = $this->publicize->get_connections( $service_name );
-								if ( empty ( $connections ) ) { ?>
-									<a id="<?php echo esc_attr( $service_name ); ?>" class="publicize-add-connection button" href="<?php echo esc_url( $connect_url ); ?>" target="_top"><?php echo esc_html( __( 'Connect', 'jetpack' ) ); ?></a>
+								if ( $connect_url ) {
+									$connections = $this->publicize->get_connections( $service_name );
+									if ( empty ( $connections ) ) { ?>
+										<a id="<?php echo esc_attr( $service_name ); ?>" class="publicize-add-connection button" href="<?php echo esc_url( $connect_url ); ?>" target="_top"><?php echo esc_html( __( 'Connect', 'jetpack' ) ); ?></a>
+									<?php } else { ?>
+										<a id="<?php echo esc_attr( $service_name ); ?>" class="publicize-add-connection button add-new" href="<?php echo esc_url( $connect_url ); ?>" target="_top"><?php echo esc_html( __( 'Add New', 'jetpack' ) ); ?></a>
+									<?php } ?>
 								<?php } else { ?>
-									<a id="<?php echo esc_attr( $service_name ); ?>" class="publicize-add-connection button add-new" href="<?php echo esc_url( $connect_url ); ?>" target="_top"><?php echo esc_html( __( 'Add New', 'jetpack' ) ); ?></a>
-			  					<?php } ?>
+									<div class="publicize-disabled-service-message">Google Plus support is being removed. <a href="#" id="jetpack-gplus-deprecated-notice" target="_blank">Why?<span class="dashicons dashicons-external"></span></a></div>
```

Comment:
This would ideally be translateable. We may also want to use "Google+" instead of "Google Plus"

Reply:
1e59e0158c96edca81f95fe5a4a2f7eee8002491 should fix this.